api/models/network/updateNetwork.js

```diff
@@ -0,0 +1,62 @@
+import { isEqual, difference, values, some } from 'lodash'
+import setupNetworkAttrs from './setupNetworkAttrs'
+
+export default function updateNetwork (userId, id, params) {
+  if (!userId) throw new Error('updateNetwork called with no userID')
+  if (!id) throw new Error('updateNetwork called with no ID')
+  return setupNetworkAttrs(userId, params).then(attrs =>
+    bookshelf.transaction(transacting =>
+      // NOTE: EnsureLoad not built to work with belongsToMany relations
```

Comment:
Hmm. It seems cleared inspecting the relevant code at the time. Mixing-in EnsureLoad and using network.ensureLoad(['communities', 'moderators']) as done in updatePost was all I was after, but it wouldn't have made much of a difference all told. I haven't grok'd yet why this isn't relevant here in the same way it was in updatePost? Doesn't matter much for now though.